pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```
@@ -1,6 +1,7 @@
 { pkgs ? import <nixpkgs> {}
 , nodejs ? pkgs.nodejs
 , yarn ? pkgs.yarn
+, allowAliases ? (pkgs.config.allowAliases or false)
```

```suggestion
, allowAliases ? pkgs.config.allowAliases or true
```

We want it to be backwards compatible by default. Only CI sets it explicitly to `true`